cli/cmd/auth/login.go

```diff
@@ -52,6 +58,38 @@ func LoginCmd(cfg *config.Config) *cobra.Command {
 			if err != nil {
 				return err
 			}
+
+			// Set default org after login
+			client, err := client.New(cfg.AdminURL, cfg.AdminToken())
+			if err != nil {
+				return err
+			}
+			defer client.Close()
+
+			orgs, err := client.ListOrganizations(context.Background(), &adminv1.ListOrganizationsRequest{})
+			if err != nil {
+				return err
+			}
+
+			if len(orgs.GetOrganization()) > 0 {
+				var orgNames []string
+				for _, org := range orgs.GetOrganization() {
+					orgNames = append(orgNames, org.Name)
+				}
+
+				defaultOrg := orgNames[0]
+				if len(orgNames) > 1 {
+					defaultOrg = cmdutil.PromptGetSelect(orgNames, "Select default org")
+				}
+
+				err = dotrill.SetDefaultOrg(defaultOrg)
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Set default organization to %q", defaultOrg)
+			}
+
```

Comment:
Move the `bold.Print("Successfully logged in.\n")` message down here (so it shows after everything is completed)